cmd/manager/manager.go

```diff
@@ -112,6 +113,9 @@ func init() {
 	managerCmd.Flags().BoolVarP(&whitelistAnnotation, "whitelist.annotations", "", false, "Require pvc whitelist annotation")
 	envs["BIVAC_WHITELIST_ANNOTATION"] = "whitelist.annotations"
 
+	managerCmd.Flags().IntVarP(&parallelCount, "parallel.count", "", 2, "The count of agents to run in parallel")
```

Comment:
Shouldn't we set the default to `1`?

Reply:
The value `2` was hardcoded before so, for me, it's fine.

Reply:
Oh, OK. I thought it was 1.